.github/workflows/test.yml

```diff
@@ -1,75 +1,43 @@
-name: Java CI with Maven
-
-on:
-  [push, pull_request]
-
-jobs:
-  unit-test:
-    strategy:
-      matrix:
-        java-version:
-          - 1.8
-          - 11
-        os:
-          - ubuntu-latest
-          - windows-latest
-          - macos-latest
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK
-        uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java-version }}
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-      - name: Unit Test
-        run: mvn -P Unittest -B verify --file pom.xml
-
-  acceptance-test:
-    needs: unit-test
-    strategy:
-      matrix:
-        splunk-version:
-          - 8.1
-          - 7.3
-    runs-on: ubuntu-latest
-    container: maven:3-jdk-8
-
-    services:
-      splunk:
-        image: splunk/splunk:${{matrix.splunk-version}}
-        env:
-          SPLUNK_START_ARGS: --accept-license
-          SPLUNK_PASSWORD: test_password
-        ports:
-          - 8089
-          - 8088
-          - 5555
-          - 15000
-          - 10667
-          - 10668/udp
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
-      - name: Cache local Maven repository
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-      - name: Acceptance Test
-        run: mvn -P AcceptanceTest -B verify --file pom.xml
-        env:
-          SPLUNK_PASSWORD: test_password
-          SPLUNK_HOST: splunk
+name: Java CI with Maven
+
+on:
+  [push, pull_request]
+
+jobs:
+
+  test-execution:
+    strategy:
+      matrix:
+        splunk-version:
+          - "latest"
+          - "8.0"
+    runs-on: ubuntu-latest
+    services:
+      splunk:
+        image: splunk/splunk:${{matrix.splunk-version}}
+        env:
+          SPLUNK_START_ARGS: --accept-license
+          SPLUNK_PASSWORD: changed!
+        ports:
+          - 8000:8000
+          - 5555:5555
+          - 8088:8088
+          - 8089:8089
+          - 15000:15000
+          - 10667:10667
+          - 10668:10668/udp
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Make Test
+        run: make test
```

Comment:
Ok, this is a bit subtle, but I'd prefer to keep the make facade as just a developer aid- internally the project should use Maven, but Make should be available and consistent across projects so a developer doesn't need to know Maven in order to launch a local build.

Reply:
Sure, no problem, the suggested changes have been made and have already been pushed, please take a look at them and let me know if any further changes are needed.